cvat-sdk/cvat_sdk/pytorch/__init__.py

```diff
@@ -135,13 +142,7 @@ def __init__(
                 f" current chunk type is {self._task.data_original_chunk_type!r}"
             )
 
```

Comment:
Probably, the function is quite long already, and could be split into several smaller ones.

Reply:
Isn't that what I'm doing in this PR? I'm factoring out a part of this function.

Reply:
Yes, right. It's a general comment, not about this PR.